cspell.json

```diff
@@ -349,7 +349,13 @@
         "zadd",
         "zstd",
         "Zulip",
-        "deser"
+        "deser",
+        "deprioritized",
+        "deprioritize",
```

Comment:
Are we teaching cspell to speak British English? :)

Reply:
I don't think we have any but I've seen "behaviour" somewhere on the list too. I don't think it matters in comments but in code I would want some consistency.